service/src/test/java/com/ericsson/eiffel/remrem/generate/integrationtest/EiffelRemremControllerIT.java

```diff
@@ -58,6 +62,10 @@
     private String activityFinishedDuplicateKeysBody;
 
     private String version = "3.0.0";
+    @Value("${erlookup.enabled}")
+    private boolean erLookUpEnabled;
+    @Value("${erlookup.url}")
```

Comment:
You could decouple "erlookup" with a dot, so it follows more Java and Spring recomendations, e.g. : "er.lookup.xxxxxxxxx"
Or if we want to be very clear, you can write whole full name for 'er':
event-repository.lookup.url , or by the way, skip the "lookup" name in the property, since its the url address to eventrepository, so use:
"event-repository.url"

Reply:
and maybe name "erlookup.enabled" property: "er.enabled" or "event-repository.enabled"

Reply:
Modified property names to  event-repository.lookup.enabled and event-repository.url as suggested.

Reply:
I thought about this one or two minutes more, and I think with should not have "lookup" in the event-repository property names,, since ER address can be used for other things in future in RemRem. So I think it should be general and just name the properties accordingly: "event-repository.enabled" and "event-repository.url". and so on, if we have more new ER properties.

Reply:
Removed "lookup" from property name . Now property names be like event-repository.enabled and event-repository.url .